accord-core/src/main/java/accord/api/Write.java

```diff
@@ -27,5 +30,6 @@
  */
 public interface Write
 {
-    void apply(Key key, Timestamp executeAt, DataStore store);
+    Future<Void> SUCCESS = ImmediateFuture.success(null);
```

Comment:
Again, not sure it is a great idea to put this in the interface, this case more so as it potentially breaks the simulator in future by pulling in implementation classes alongside interface classes.